dev-docs/modules/floors.md

```diff
@@ -636,7 +636,24 @@ Priority order for three column rule sets:
  \* \| \* \| \_  
  \* \| \* \| \*  
 
-
+Priority order for four column rule sets:
+
+\_ \| \_ \| \_ \| \_
+\_ \| \_ \| \_ \| \*
+\_ \| \_ \| \* \| \_
+\_ \| \* \| \_ \| \_
+\* \| \_ \| \_ \| \_
+\_ \| \_ \| \* \| \*
+\_ \| \* \| \_ \| \*
+\* \| \_ \| \_ \| \*
+\_ \| \* \| \* \| \_
```

Comment:
648 and 649 should be swapped according to how the logic currently works:

Here is an example of what the code actually will generate (Highest index in array is selected first)

```
[
    "_ | _ | _ | _",
    "_ | _ | _ | *",
    "_ | _ | * | _",
    "_ | * | _ | _",
    "* | _ | _ | _",
    "_ | _ | * | *",
    "_ | * | _ | *",
    "_ | * | * | _",
    "* | _ | _ | *",
    "* | _ | * | _",
    "* | * | _ | _",
    "_ | * | * | *",
    "* | _ | * | *",
    "* | * | _ | *",
    "* | * | * | _",
    "* | * | * | *"
]
```